build.gradle.kts

```diff
@@ -3,8 +3,8 @@ import java.time.LocalDateTime
 plugins {
     application
     java
-    id("org.openjfx.javafxplugin") version "0.0.13"
-    id("org.beryx.jlink") version "2.25.0"
+    id("org.openjfx.javafxplugin") version "0.0.14"
+    id("org.beryx.jlink") version "3.1.1"
     id("com.diffplug.spotless") version "6.11.0"
```

Comment:
Also need to update spotless to 6.19  or later, or set the `googleJavaFormat()` version to 1.17.0 or later.

Reply:
Bumped spotless version to 6.25.0 (same as MapTool)